### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by madig**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
```
Comment (anthrotype) [inline .editorconfig]: we don't need this if we are going to autoformat with black

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  rev: v2.1.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-ast
+  - id: mixed-line-ending
+    args: ["--fix=lf"]
+  - id: debug-statements
+  - id: flake8
```
Comment (anthrotype) [inline .pre-commit-config.yaml]: flake8 official repo now exports its own pre-commit hook, e.g. see https://github.com/asottile/pyupgrade/pull/95

File: .editorconfig (hunk 1 above)
Comment (madig) [reply .editorconfig]: It helps VSCode do the right thing. I tried.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (madig) [reply .pre-commit-config.yaml]: Oh nice. Can't find an immediate mention of bugbear but it doesn't seem to complain about the source.

**Pull request merged**